package.json

```diff
@@ -308,5 +308,8 @@
 			"pre-commit": "npm run -s install-if-no-packages && node bin/pre-commit-hook.js",
 			"pre-push": "npm run -s install-if-no-packages && node bin/pre-push-hook.js"
 		}
+	},
+	"devDependencies": {
+		"@signal-noise/stylelint-scales": "1.2.0"
```

Comment:
I think our Calypso monorepo setup was preventing us from defining dependencies as `devDependencies` and we were always forced to add them as `dependencies`. Have this changed @Automattic/team-calypso ?

Reply:
Good catch @mmtr, I didn't notice this was the only item in devDependencies! @Automattic/team-calypso is there some documentation on this that we could read up on?

Reply:
🤔 Since stylelint is in dependencies, I guess we'd move it there? 

There's a broad overview in the following readme, but I didn't spot anything for build or dev deps.
https://github.com/Automattic/wp-calypso/blob/master/docs/monorepo.md

Reply:
Adding it to `dependencies` is the way to go, yes.  In the context of the root of the monorepo where you do your actual installs, there really isn't a practical difference between `dependencies` and `devDependencies`.  In that context, "dev" is really the overall purpose of the install.